runtime/tests/tests.rs

```diff
@@ -164,6 +219,7 @@ fn reward_validators(validators: &[(AccountId, AccountId)]) {
 #[test]
 fn start_session_works() {
 	ExtBuilder::default().build().execute_with(|| {
+		// TODO test the current session index after each set
```

Comment:
Remove this TODO, at the end of `start_session` function already do this

Reply:
Yeah, I see. I would have written it differently myself. In any test, the expectations should be explicit. But not a big issue here. I will remove the comment.

Reply:
Done